in_toto/resolvers/github.py

```diff
@@ -0,0 +1,37 @@
+from urllib3.util import Url
+from urllib3 import PoolManager
+import json
+import hashlib
+
+def get_resolvable_url(generic_url):
+    #TODO: get resolvable url for commits, release, etc.
```

Comment:
We should consider what the generic_url looks like. The format is defined by what the resolver expects, essentially, and it can't be just the path of the API. There's some guidance regarding this in ITE-4.